src/Diagnostics.RuntimeHost/Controllers/DiagnosticControllerBase.cs

```diff
@@ -865,8 +865,7 @@ private List<DataProviderMetadata> GetDataProvidersMetadata(DataProviders.DataPr
             {
                 if (dataProvider.FieldType.IsInterface)
                 {
-                    var metadataProvider = dataProvider.GetValue(dataProviders) as IMetadataProvider;
-                    if (metadataProvider != null)
+                    if (dataProvider.GetValue(dataProviders) is IMetadataProvider metadataProvider)
```

Comment:
What changes between `as` and `is`?

Reply:
This is visual studio default refactoring suggestion :-)

Instead of two lines, it is suggesting to combine null check and typecasting to IMetadataProvider in one line